src/enums/summaryActions.ts

```diff
@@ -0,0 +1,5 @@
+export enum SummaryActions {
+  SAVE_RESUME_LATER,
+  DELETE_ALL,
+  CONFIRM
+}
```

Comment:
FYI, if you don't assign values to enums then they are automatically numbered (starting from 0).